### PR TITLE
fix: remove skipped doc links

### PR DESCRIPTION
**Pull request by dvilelaf**
## Proposed changes

Remove skipped link checks for CI testing purposes.
Investigating [this](https://github.com/valory-xyz/open-autonomy/actions/runs/3288400682/jobs/5418997739) failure that was passing locally:
```
Awaiting for results...
Found broken url in the docs:
docs/get_started/why_do_we_need_agent_services.md: ['https://encyclopedia.pub/entry/2959']
ERROR: InvocationError for command /home/runner/work/open-autonomy/open-autonomy/scripts/check_doc_links.py (exited with code 1)
```

Redirects should not be a problem as the [requests library automatically follows](https://requests.readthedocs.io/en/latest/user/quickstart/#redirection-and-history) those. Could it be that we were getting a TooManyRedirects and being rate-limited? I've added the http status code to the logging so it is easier to debug and fix next time.

## Fixes

n/a

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [x] Non-breaking fix (non-breaking change which fixes an issue)
- [ ] Breaking fix (breaking change which fixes an issue)
- [ ] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [ ] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [x] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have locally run services that could be impacted and they do not present failures derived from my changes

### CODE REVIEW

File: scripts/check_doc_links.py [hunk 1]
```diff
@@ -141,7 +145,7 @@ def main() -> None:  # pylint: disable=too-many-locals
         if broken_links:
             broken_links_str = "\n".join(
                 [
-                    f"{file_name}: {[url[1] for url in urls]}"
+                    f"{file_name}: {[(url[1], url[2]) for url in urls]}"
```
Comment (dvilelaf) [inline scripts/check_doc_links.py:148]: Show the status code next time it fails.

Comment (Adamantios) [reply scripts/check_doc_links.py:172]: Future devs and self would need to study the `broken_links` in order to understand what these indexes are accessing. A `dataclass` or a `dict` would have been more helpful. However, we could at least add a comment on the code with what these indexes are accessing.

**Review by Adamantios — DISMISSED** [review]

File: scripts/check_doc_links.py (hunk 1 above)
Comment (dvilelaf) [reply scripts/check_doc_links.py]: Fixed

**Review by Adamantios — APPROVED** [review]

File: scripts/check_doc_links.py [hunk 2]
```diff
@@ -90,29 +131,18 @@ def main() -> None:  # pylint: disable=too-many-locals
     """Check for broken or HTTP links"""
     all_md_files = [str(p.relative_to(".")) for p in Path("docs").rglob("*.md")]
 
-    broken_links: Dict[str, List[str]] = {}
+    broken_links: Dict[str, Dict] = {}
```
Comment (Karrenbelt) [inline scripts/check_doc_links.py:134]: next time please specify type inside nested `Dict` as well

**Pull request merged**